<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompanyView</class>
 <widget class="QWidget" name="CompanyView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="listWidgetPackage"/>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Recipient</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditRecipient"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Street</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEditStreet"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Location</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditLocation"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonAdd">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonAdd</sender>
   <signal>clicked()</signal>
   <receiver>CompanyView</receiver>
   <slot>addPackage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>429</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addPackage()</slot>
 </slots>
</ui>
